src/cdn.ts

```diff
@@ -1,2 +1,81 @@
-import { startTestBox } from ".";
-startTestBox(window.__tbxConfig || { allowFullStory: true });
+import { registerLoginHandler, startTestBox } from ".";
+
+const params = new URLSearchParams(window.location.search);
+const testcase = params.get("testcase");
+
+async function fakeLoginHandler(props) {
+  console.log("PROPS", props);
+  return true;
+}
+
+switch (testcase) {
+  case "delayed-login-handler-register":
+    startTestBox(window.__tbxConfig || { allowFullStory: true });
+    setTimeout(() => registerLoginHandler(fakeLoginHandler), 3000);
+    postMessage({
+      testbox: {
+        version: 1,
+        sender: "app",
+        event: "login",
+        data: {
+          email: "testuser1@tbxofficial.com",
+          password: "password",
+          first_name: "Test",
+          last_name: "User 1",
+        },
+      },
+    });
+    postMessage({
+      testbox: {
+        version: 1,
+        sender: "app",
+        event: "login",
+        data: {
+          email: "testuser2@tbxofficial.com",
+          password: "password",
+          first_name: "Test",
+          last_name: "User 2",
+        },
+      },
+    });
```

Comment:
No need for two login messages in this test case. Only one should be send by core.